src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DataContractSerializer.cs

```diff
@@ -32,6 +32,18 @@ public sealed class DataContractSerializer : XmlObjectSerializer
 
         private static SerializationOption s_option = IsReflectionBackupAllowed() ? SerializationOption.ReflectionAsBackup : SerializationOption.CodeGenOnly;
         private static bool s_optionAlreadySet;
+
+        internal static readonly UTF8Encoding SafeUTF8 = new UTF8Encoding(false, false);
+        internal static readonly UTF8Encoding ValidatingUTF8 = new UTF8Encoding(false, true);
+
+        internal static readonly UnicodeEncoding SafeUTF16 = new UnicodeEncoding(false, false, false);
+        internal static readonly UnicodeEncoding SafeBEUTF16 = new UnicodeEncoding(true, false, false);
+        internal static readonly UnicodeEncoding ValidatingUTF16 = new UnicodeEncoding(false, false, true);
+        internal static readonly UnicodeEncoding ValidatingBEUTF16 = new UnicodeEncoding(true, false, true);
+
+        internal static readonly Base64Encoding Base64Encoding = new Base64Encoding();
+        internal static readonly BinHexEncoding BinHexEncoding = new BinHexEncoding();
```

Comment:
Nit: we try to use s_ for static fields.

Reply:
changing to properties